modules/web/src/assets/config/changelog.json

```diff
@@ -1,161 +1,173 @@
 {
-  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.23.md",
+  "externalChangelogURL": "https://github.com/kubermatic/kubermatic/blob/main/docs/changelogs/CHANGELOG-2.24.md",
   "entries": [
     {
-      "category": "added",
-      "description": "Add support for Kubernetes 1.27."
+      "category": "security",
+      "description": "Update Cilium to 1.14.3 and 1.13.8 and deprecate older patch releases (fixes CVE-2023-44487, CVE-2023-39347, CVE-2023-41333, CVE-2023-41332). Updating user clusters is strongly recommended."
     },
     {
-      "category": "added",
-      "description": "Add support for Kubernetes 1.24.15, 1.25.11, 1.26.6 and 1.27.3 (fixing CVE-2023-2431, CVE-2023-2727 and CVE-2023-2728)."
+      "category": "security",
+      "description": "Update curl to 8.4.0 (fixes CVE-2023-38545, CVE-2023-38546). curl is only available for usage in the optional web terminal feature."
     },
     {
       "category": "added",
-      "description": "Add option to disable deployment of default network policies in KubeVirt cluster."
+      "description": "EE only: Add KubeLB integration."
     },
     {
       "category": "added",
-      "description": "Add support for Canal 3.25."
+      "description": "Support for Kubernetes 1.28."
```

Comment:
It seems like we already got this covered in the note below. Should we remove this?

Reply:
Fair point. Let me remove it!

Reply:
Done.